Building static pure binary fails

I am trying to build a pure, static go binary using `bazel build //binary:binary --features=static --features=pure` with following target

```
go_binary(
    name = "binary",
    embed = [":go_default_library"],
    visibility = ["//visibility:public"],
)
```

and the go library can be anything from a simple file printing hello world to a more complex library with external dependencies.

I am getting following error:

```
(15:32:02) ERROR: /home/user/Projects/name/core2/binary/BUILD.bazel:22:1: GoLink binary/linux_amd64_static_pure_stripped/binary failed (Exit 1): link failed: error executing command 
  (cd /home/user/.cache/bazel/_bazel_user/5722e101c579002746f733c77caf52d7/execroot/__main__ && \
  exec env - \
  bazel-out/host/bin/external/io_bazel_rules_go/go/tools/builders/linux_amd64_static_pure_stripped/link -go external/go_sdk/bin/go -root_file bazel-out/k8-fastbuild/bin/external/go_stdlib_linux_amd64_pure/ROOT -goos linux -goarch amd64 '-cgo=0' -compiler_path /usr/bin -cc /usr/bin/gcc -cpp_flag -U_FORTIFY_SOURCE -cpp_flag -fstack-protector -cpp_flag -B/usr/bin -cpp_flag -B/usr/bin -cpp_flag -Wunused-but-set-parameter -cpp_flag -Wno-free-nonheap-object -cpp_flag -fno-omit-frame-pointer -ld_flag -lstdc++ -ld_flag -lm -ld_flag '-fuse-ld=gold' -ld_flag -Wl,-no-as-needed -ld_flag -Wl,-z,relro,-z,now -ld_flag -B/usr/bin -ld_flag -B/usr/bin -ld_flag -pass-exit-codes -L . -L 'bazel-out/k8-fastbuild/bin/external/org_golang_x_sys/unix/linux_amd64_static_pure_stripped/go_default_library~/' -L 'bazel-out/k8-fastbuild/bin/external/org_golang_x_crypto/ssh/terminal/linux_amd64_static_pure_stripped/go_default_library~/' -L 'bazel-out/k8-fastbuild/bin/external/com_github_sirupsen_logrus/linux_amd64_static_pure_stripped/go_default_library~/' -L 'bazel-out/k8-fastbuild/bin/common/go/env/linux_amd64_static_pure_stripped/go_default_library~/' -L 'bazel-out/k8-fastbuild/bin/external/com_github_go_chi_chi/linux_amd64_static_pure_stripped/go_default_library~/' -L 'bazel-out/k8-fastbuild/bin/external/com_github_go_chi_chi/middleware/linux_amd64_static_pure_stripped/go_default_library~/' -L 'bazel-out/k8-fastbuild/bin/common/go/middleware/linux_amd64_static_pure_stripped/go_default_library~/' -L 'bazel-out/k8-fastbuild/bin/external/com_github_lib_pq/oid/linux_amd64_static_pure_stripped/go_default_library~/' -L 'bazel-out/k8-fastbuild/bin/external/com_github_lib_pq/linux_amd64_static_pure_stripped/go_default_library~/' -L 'bazel-out/k8-fastbuild/bin/binary/linux_amd64_static_pure_stripped/binary~/' -- -o bazel-out/k8-fastbuild/bin/binary/linux_amd64_static_pure_stripped/binary -linkmode external -w -extld /usr/bin/gcc -extldflags '-U_FORTIFY_SOURCE -fstack-protector -Wall -B/usr/bin -B/usr/bin -Wunused-but-set-parameter -Wno-free-nonheap-object -fno-omit-frame-pointer -fno-canonical-system-headers -Wno-builtin-macro-redefined -D__DATE__="redacted" -D__TIMESTAMP__="redacted" -D__TIME__="redacted" -lstdc++ -lm -fuse-ld=gold -Wl,-no-as-needed -Wl,-z,relro,-z,now -B/usr/bin -B/usr/bin -pass-exit-codes -Wl,-rpath,$ORIGIN/../../ -static' 'bazel-out/k8-fastbuild/bin/binary/linux_amd64_static_pure_stripped/binary~/gitlab.com/namebank/core2/binary.a')

Use --sandbox_debug to see verbose messages from the sandbox
/home/user/.cache/bazel/_bazel_user/5722e101c579002746f733c77caf52d7/bazel-sandbox/3141159285248290298/execroot/__main__/bazel-out/k8-fastbuild/bin/external/go_stdlib_linux_amd64_pure/pkg/tool/linux_amd64/link: running /usr/bin/gcc failed: exit status 1
/home/user/.cache/bazel/_bazel_user/5722e101c579002746f733c77caf52d7/bazel-sandbox/3141159285248290298/execroot/__main__/tmp/go-link-383460772/go.o:go.go:_cgo_callers: error: undefined reference to 'x_cgo_callers'
/home/user/.cache/bazel/_bazel_user/5722e101c579002746f733c77caf52d7/bazel-sandbox/3141159285248290298/execroot/__main__/tmp/go-link-383460772/go.o:go.go:_cgo_init: error: undefined reference to 'x_cgo_init'
/home/user/.cache/bazel/_bazel_user/5722e101c579002746f733c77caf52d7/bazel-sandbox/3141159285248290298/execroot/__main__/tmp/go-link-383460772/go.o:go.go:_cgo_mmap: error: undefined reference to 'x_cgo_mmap'
/home/user/.cache/bazel/_bazel_user/5722e101c579002746f733c77caf52d7/bazel-sandbox/3141159285248290298/execroot/__main__/tmp/go-link-383460772/go.o:go.go:_cgo_munmap: error: undefined reference to 'x_cgo_munmap'
/home/user/.cache/bazel/_bazel_user/5722e101c579002746f733c77caf52d7/bazel-sandbox/3141159285248290298/execroot/__main__/tmp/go-link-383460772/go.o:go.go:_cgo_notify_runtime_init_done: error: undefined reference to 'x_cgo_notify_runtime_init_done'
/home/user/.cache/bazel/_bazel_user/5722e101c579002746f733c77caf52d7/bazel-sandbox/3141159285248290298/execroot/__main__/tmp/go-link-383460772/go.o:go.go:_cgo_sigaction: error: undefined reference to 'x_cgo_sigaction'
/home/user/.cache/bazel/_bazel_user/5722e101c579002746f733c77caf52d7/bazel-sandbox/3141159285248290298/execroot/__main__/tmp/go-link-383460772/go.o:go.go:_cgo_thread_start: error: undefined reference to 'x_cgo_thread_start'
/home/user/.cache/bazel/_bazel_user/5722e101c579002746f733c77caf52d7/bazel-sandbox/3141159285248290298/execroot/__main__/tmp/go-link-383460772/go.o:go.go:runtime._cgo_setenv: error: undefined reference to 'x_cgo_setenv'
/home/user/.cache/bazel/_bazel_user/5722e101c579002746f733c77caf52d7/bazel-sandbox/3141159285248290298/execroot/__main__/tmp/go-link-383460772/go.o:go.go:runtime._cgo_unsetenv: error: undefined reference to 'x_cgo_unsetenv'
/home/user/.cache/bazel/_bazel_user/5722e101c579002746f733c77caf52d7/bazel-sandbox/3141159285248290298/execroot/__main__/tmp/go-link-383460772/go.o:go.go:runtime.cgo_yield: error: undefined reference to '_cgo_yield'
collect2: error: ld returned 1 exit status

2018/01/22 15:32:02 error running linker: exit status 1
```

Just building with `--features=pure` or `--features=static` by themselves works.

--- a/go/private/actions/link.bzl
+++ b/go/private/actions/link.bzl
@@ -56,7 +56,6 @@ def emit_link(go,
   if go.mode.msan:
     gc_linkopts.append("-msan")
   if go.mode.static:
-    link_external = True
     extldflags.append("-static")
   if go.mode.link != LINKMODE_NORMAL:
     gc_linkopts.extend(["-buildmode", go.mode.link])

